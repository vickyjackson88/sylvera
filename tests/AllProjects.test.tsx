import { render } from "@testing-library/react";

import AllProjects from "../pages/projects/all";

const mockProjects = ["project1", "project2", "project3"];

describe("Snapshot tests: projects/all", () => {
  it("renders AllProjects DOM unchanged", () => {
    const { container } = render(<AllProjects projects={mockProjects} />);
    expect(container).toMatchSnapshot();
  });
});
