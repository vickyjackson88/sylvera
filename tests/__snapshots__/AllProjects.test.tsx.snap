// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot tests: projects/all renders AllProjects DOM unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <h1
      class="title"
    >
      Live Projects
    </h1>
    <div
      class="card"
    >
      <a
        class="link"
        data-testid="project1-link"
        href="/projects/project1"
      >
        <span>
          project1
        </span>
        <span>
          →
        </span>
      </a>
    </div>
    <div
      class="card"
    >
      <a
        class="link"
        data-testid="project2-link"
        href="/projects/project2"
      >
        <span>
          project2
        </span>
        <span>
          →
        </span>
      </a>
    </div>
    <div
      class="card"
    >
      <a
        class="link"
        data-testid="project3-link"
        href="/projects/project3"
      >
        <span>
          project3
        </span>
        <span>
          →
        </span>
      </a>
    </div>
  </div>
</div>
`;
